src/commands/check.rs

```diff
@@ -37,9 +52,47 @@ pub struct Common {
         ignore_case = true,
         action = clap::ArgAction::Append,
         num_args = 1,
+        hide_possible_values = true,
     )]
     pub except: Vec<String>,
 
+    /// Enable all lint rules. This includes additional rules outside the
+    /// default set.
+    ///
+    /// `--except <RULE>` can be used in conjuction with this argument. Using
+    /// this argument will ignore certain configuration file options.
+    #[clap(short, long, conflicts_with_all = ["include_lint_tag", "exclude_lint_tag"])]
+    pub all_lint_rules: bool,
+
+    /// Excludes a lint tag from running. This implies all other lint tags
```

Comment:
This says `all`. I'm not sure how to make it clearer, but I think this means that it will run all other rules that are otherwise enabled (which may be the default set or any other set).

Reply:
I'm working on changing this to act as a filter on the other selections (via `only_lint_tag` and `all_lint_rules`). Should have a change pushed shortly